templates/default/unzip-hdfs-files.sh.erb

```diff
@@ -59,7 +59,7 @@ echo "UNZIPPING" > "$fsmFile"
 
 
 cd "$workdir"
-<%= @dtrx %> -o -n "$file"
+dtrx -o -n "$file"
 check_error
```

Comment:
@robzor92 This was templated by @jimdowling in its last PR. Not completely sure why. But if you think that the template is not necessary, you should also remove the part in install recipe.

Reply:
Aha, the problem was that this is not valid syntax. It resolved to empty string in the script which broke everything. @jimdowling what's the intention behind this templating?

Reply:
There is a reason for the templating. dtrx is not on the $PATH in centos - we install it from source, and it ends up in /usr/local/bin/dtrx
In ubuntu, it's installed as a package (probably in /usr/bin/dtrx).

Reply:
From hopsworks::install.rb, you will see:
'dtrx' should not be an empty string.


install.rb:  package "dtrx"
install.rb:  dtrx="dtrx"
install.rb:  remote_file "#{Chef::Config['file_cache_path']}/dtrx.tar.gz" do
install.rb:    source node['download_url'] + "/#{node['dtrx']['version']}"
install.rb:  bash "unpack_dtrx" do
install.rb:    tar -xzf dtrx.tar.gz
install.rb:    cd dtrx-7.1
install.rb:    # dtrx expects 7z to on its path. create a symbolic link from /bin/7z to /bin/7za
install.rb:    not_if "which dtrx"
install.rb:  dtrx="/usr/local/bin/dtrx"
install.rb:              :dtrx => dtrx